### DISCUSSION TITLE
cheats doesn't work v1.0.0

### DISCUSSION THREAD

**Discussion by sirdrakley** [GoldHEN/GoldHEN_Cheat_Repository#71] [Q&A]
Hi! 
Kingdom Hearts 2 Final mix cheats doesn't work. Please fix them. 
Thanks

**Comment by kmeps4** [comment]
I need the Cheat CUSA_ID and version please!

**Reply by sirdrakley** [reply]
Kingdom Hearts 2 Final mix cheats doesn't work. Please fix them. 
CUSA05786 – EUR
Update 1.04

Thanks! greetings from Spain:)
